src/main/java/com/dumbdogdiner/stickyapi/bukkit/user/StickyUserBukkit.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2020 DumbDogDiner <a href="dumbdogdiner.com">&lt;dumbdogdiner.com&gt;</a>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.bukkit.user;
+
+import com.dumbdogdiner.stickyapi.common.user.StickyUser;
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.Sound;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.UUID;
+
+public class StickyUserBukkit extends StickyUser {

```

Comment:
I again ask why this is necessary when we can exchange data via plugin APIs - persistence for offline players?